deploy/gk-deploy

```diff
@@ -40,20 +43,26 @@ Arguments:
 Options:
   -g, --deploy-gluster
               Deploy GlusterFS pods on the nodes in the topology that contain
-              brick devices. Default is to not deploy GlusterFS, thus assuming
-              that has already been done.
+              brick devices. If the --abort flag is also specified, this flag
+              indicates that all GlusterFS pods and deployments should be
+              deleted as well. Default is to not handle GlusterFS deployment
+              or removal.
 
   -t TEMPLATES, --templates_dir TEMPLATES
               Location of directory containing the heketi templates for
               OpenShift. Default is '${TEMPLATES}'.
 
+  -n NAMESPACE, --namespace NAMESPACE
+              The namespace to use for creating resources.
+              Defaults is '${NAMESPACE}'.
+
   -w SECONDS, --wait SECONDS
               Wait SECONDS seconds for pods to become ready. Default is '${WAIT}'.
 
   -l LOG_FILE, --log LOG_FILE
               Save all output to the specified file.
 
-  --abort     Abort a failed deployment. WARNING: Deletes all related resources.
+  --abort     Abort a deployment. WARNING: Deletes all related resources.
```

Comment:
Instead of a complete removal, it would good to give an option or checkpoint based removal. For ex: if the gluster pods are up and running, we may not delete those and recreate it.. Thus we could have different checkpoints and receive inputs from the user and proceed. This could help us to rollback to a particular state and proceed from there.